### ISSUE TITLE
Payment Integration

### ISSUE THREAD

**Issue by dyjo** [Sylius/Sylius#48]
Hello,

First: Thank you for the awesome library! 

Sylius would be perfect in a product I have scheduled for imminent release, but I was confused about some things regarding the payment process. I'm hoping to figure out what's missing so I can bridge the gap for my project, and submit some pull requests if the code might help others.
- **Is Omnipay missing from the standard distro?** The two options set in sylius/app/config/sylius.yml are 'dummy' and 'stripe', which are names of omnipay gateways. But neither the omnipay library nor `sylius/omnipay-bundle` is included in `sylius/sylius` or `sylius/payments-bundle`.
- _(related)_ **Is the CreditCard From Type missing?** I see the `PaymentStep` in the Checkout process and the `PaymentMethodType` in the PaymentBundle. But I can't seem to find how the actual entry of credit card information is facilitated? Is this a work in progress related to the integration of the Omnipay library?
- **What else?** I've only started reading the source in the past day, so I'm sure there are things I've missed or overlooked. Not to say "hurry up and finish!", only that it would be nice to have a list of things that need to be done before using the lib in its current state. I.e. "If you plan to use Sylius in a live project, you will have to do X, Y, Z."

Again, thanks for the awesome library; I look forward to seeing what you come up with next, and hope to have an opportunity to contribute in the future!
- dyjo
